docs/upgrading/splinter-v0.3.12-from-v0.3.11.md

```diff
@@ -209,3 +210,32 @@ building pattern that's enabled by the Sabre, Sawtooth, and Transact SDKs. This
 is the recommended approach because it greatly simplifies the process of
 creating Sabre transactions and batches. For an example of this pattern, see the
 [scabbard CLI](https://github.com/Cargill/splinter/blob/v0.3.12/services/scabbard/src/cli/main.rs)
+
+## The scabbard CLI's upload command has been changed
+In v0.3.11, you could upload a smart contract to scabbard using the experimental
```

Comment:
Can you add a line before this that says "If you were using the experimental scabbard CLI to upload smart contracts, you must update your procedure"

Reply:
Added